config/initializers/two_factor_auth_controller_override.rb

```diff
@@ -18,6 +18,10 @@ def handle_two_factor_authentication
         end
       end
 
+      def teaspoon_controller?
```

Comment:
oh if we want this for development too, maybe we could do it in an initialzier, and guard on env:

```ruby
# config/initializers/teaspoon.rb
if !Rails.env.production?
  class Teaspoon::SuiteController
    skip_before_filter :handle_two_factor_authentication
  end
end
```